continuous_integration/install.sh

```diff
@@ -33,6 +33,7 @@ conda install --yes numpy scipy matplotlib netcdf4 nose sphinx numpydoc hdf4=4.2
 conda install --yes sphinx_rtd_theme
 
 
+pip install git+https://github.com/jleinonen/pytmatrix.git
```

Comment:
My only comment is if the pytmatrix is broken, this also breaks pydisdrometer. Not sure there is any better method though...